### PR TITLE
adding maddj recommendations and a delightful .gitignore file...

### LINKED ISSUE #5: maddj's recommendations for things to do
# Things
- [ ] LIST OF ACTIVITIES
- [ ] LIST OF FOODS

### PR DESCRIPTION
**Pull request by maddj**
blah blah blah.
more stuff.
blah!
⚠️ 😝 
Closes #5

### CODE REVIEW

File: maddj_recs.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Recs for Los Angeles!
+
+### THINGS to DO
+* Petersen Auto Museum!
+
+### FOODS to EAT
```
Comment (al-niessner) [inline maddj_recs.md:7]: Oh, here some feedback for a super simple example that is going to be a single comment.

File: maddj_recs.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## Recs for Los Angeles!
+
+### THINGS to DO
```
Comment (al-niessner) [inline maddj_recs.md:3]: A review...

**Review by al-niessner — CHANGES_REQUESTED** [review]

File: maddj_recs.md (hunk 1 above)
Comment (maddj) [reply maddj_recs.md:7]: good point and such

**Review by al-niessner — APPROVED** [review]

File: maddj_recs.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+## Recs for Los Angeles!
+
```
Comment (al-niessner) [inline maddj_recs.md:2]: More review

**Review by al-niessner — CHANGES_REQUESTED** [review]

**Review by al-niessner — APPROVED** [review]

**Pull request merged**